src/render.ts

```diff
@@ -0,0 +1,21 @@
+import { type TemplateElement } from './view/abstract-element';
+
+enum RenderPosition {
+  BEFOREBEGIN = 'beforebegin',
+  AFTERBEGIN = 'afterbegin',
+  BEFOREEND = 'beforeend',
+  AFTEREND = 'afterend',
+}
+
+function createElement(template: string): Element {
+  const newElement: HTMLDivElement = document.createElement('div');
+  newElement.innerHTML = template;
+
+  return newElement.firstElementChild as Element;
+}
```

Comment:
Я бы посоветовал сюда Generic. Нам может понадобиться создавать не абстрактный элемент, а что-то более специфичное (кнопку, ссылку и т.д.). И в качестве дефолта див, потому что самый частый кейс
```suggestion
function createElement<E extends Element = HTMLDivElement>(template: string): E {
  const newElement: HTMLDivElement = document.createElement('div');
  newElement.innerHTML = template;

  return newElement.firstElementChild as E;
}
```